default.vcl

```diff
@@ -26,6 +26,10 @@ sub vcl_recv {
     elif (req.url ~ "^\/content\/feed\/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}.*$") {
         set req.backend_hint = dynBackend.backend("ccf-content-rss");
     }
+    elif(req.url ~ "^\/content\/advanced-search.*$") {
+        set req.url = "/search";
```

Comment:
_Might_ be more future proof if we use the following but the current version is also OK.
```suggestion
        set req.url = regsub(req.url, "^\/content\/advanced-(.*)$", "/\1");
```